tests/foreman/cli/test_acs.py

```diff
@@ -0,0 +1,277 @@
+"""Test class for Alternate Content Sources CLI
+
+:Requirement: AlternateContentSources
+
+:CaseAutomation: Automated
+
+:CaseLevel: Component
+
+:CaseComponent: AlternateContentSources
+
+:Team: Phoenix
```

Comment:
@jyejare @vsedmik Can you correct the team name here, it should be `phoenix-content`